mapreduce/README.md

```diff
@@ -0,0 +1,93 @@
+# MapReduce-TiDB-Connector
+
+## Build
+
+```bash
+git clone git@github.com:pingcap-incubator/TiBigData.git
+cd TiBigData
+mvn clean package -DskipTests -am -pl mapreduce/mapreduce-base
+cp mapreduce/mapreduce-2.0/target/mapreduce-tidb-connector-base-0.0.4-SNAPSHOT.jar ${HOME}/lib
+```
+
+## Version
+
+`hadoop 2.X` and `hadoop 3.X` are supported.
+
+
+## Demo
+
+```bash
+hadoop jar mapreduce-tidb-connector-base-0.0.4-SNAPSHOT.jar io.tidb.bigdata.mapreduce.tidb.examples.TiDBMapreduceDemo  \
+ -du  ${DATABASE_URL} \
+ -u   ${USERNAME} \
+ -p   ${PASSWORD}  \
+ -dn  ${TIDB_DATABASE} \
+ -n   ${TABLE_NAME} \
+ -f   ${FIELD1} \
+ -f   ${FIELD2} \
+ -t   ${TABLE_NAME}
+ -ts  ${SNAPSHOT_TIMESTAMP} \
+ -l   ${LIMIT} 
+```
+### example
+
```

Comment:
Step 1: create table ...

Reply:
already add create table and insert record sql